modules/common/react/lib/styles/focusRing.ts

```diff
@@ -1,29 +1,19 @@
 import {keyframes, CSSObject} from '@emotion/core';
 import canvas from '@workday/canvas-kit-react-core';
-import {CanvasTheme} from '@workday/canvas-kit-labs-react-core';
+import {CanvasTheme, defaultCanvasTheme} from '@workday/canvas-kit-labs-react-core';
 import memoize from 'lodash/memoize';
 
-type FocusRingParams = {
-  ringWidth: number;
-  separationWidth: number;
-  animate: boolean;
-  inset: boolean;
-  innerShadowColor: string;
-  outerShadowColor: string;
-};
-
 function calculateFocusRing({
-  ringWidth,
-  separationWidth,
+  width = 2,
+  separation = 0,
```

Comment:
Why are defaults needed here? They will always be defined coming from `focusRing` since it has its own defaults